$NetBSD: patch-ab,v 1.1 2008/02/16 15:21:40 heinz Exp $

  Allow installation to DESTDIR (taken from safecat-buildroot.patch in
  the archive).

--- Makefile.orig	2004-08-17 02:50:09.000000000 +0200
+++ Makefile
@@ -31,7 +31,7 @@ compile auto-str.c substdio.h readwrite.
 
 auto_home.c: \
 auto-str conf-root
-	./auto-str auto_home `head -1 conf-root` > auto_home.c
+	./auto-str auto_home $(DESTDIR)`head -1 conf-root` > auto_home.c
 
 auto_home.o: \
 compile auto_home.c
